pc/PlatformDeviceControl.c

```diff
@@ -35,9 +35,14 @@ int usbFdWrite = -1;
 int usbFdRead = -1;
 #endif  /*USE_USB_VSC*/
 
+#include "XLinkPublicDefines.h"
+
 #define USB_LINK_SOCKET_PORT 5678
 #define UNUSED __attribute__((unused))
 
+
+static UsbSpeed_t usb_speed_enum = X_LINK_USB_SPEED_UNKNOWN;
+static char mx_serial[128] = { 0 };
```

Comment:
change to `static char mx_serial[XLINK_MAX_MXID] = { 0 };` as Martin suggested @saching13